.github/workflows/ci.yml

```diff
@@ -42,7 +42,4 @@ jobs:
       - run: npm install
       - run: npm test
       - run: npm i codecov
```

Comment:
Wait a second - why do you need to install this here?  Instead, should we make sure it's in our package.jsons?

Reply:
It seems like it'd be okay since it's the repo bot that's trying to run codecov, and not the individual repo `package.json`. I guess the place that'd be problematic is if we took the output of this process and ran other things that ended up depending on it.

Reply:
So technically, the `codecov` command line tool could have breaking changes from major version to major version.  I don't think it's likely, but it could happen.  I'd like to think we should either a.) include the dependency in package.json or b.) specify the version in the `npm i` call here.

If we go with `b`, may make sense to use `npx` too :)